### DISCUSSION TITLE
Add CMI Skins support to ItemsAdder.

### DISCUSSION THREAD

**Discussion by Distolfix** [Zrips/CMI#7656] [Enhancement]
Dear, the CMI plugin for the Skin module makes the use of Items adder incompatible. As we can see at the following link. The creator of the plugin offers a method to fix it with SkinsRestorer.
https://itemsadder.devs.beer/compatibility-with-other-plugins/compatible/skinsrestorer
But it doesn't offer a possibility to make it compatible with your cmi skin module. I also noticed that SkinsRestorer has also added the possibility to fix this problem from the config. So I would like to propose to fix this problem as I am very happy with your Skin module. From what I understand as soon as you enter, if the skin is applied by default it does not allow you to download the resourcepack.

I hope you will listen to me and fix it soon. As I had to switch to skinsrestorer to get around this problem. and I hope to use your skin module again

- Francesco

**Comment by nuoxi4n** [comment]
Support
